README.md

```diff
@@ -1,27 +1,13 @@
-# VirtrolioSite
+<img align="left" src="https://github.com/virtrolio/virtrolio-site/blob/master/src/assets/images/logo_reg.png" width="200px" height="200px">
 
-This project was generated with [Angular CLI](https://github.com/angular/angular-cli) version 9.1.6.
+# Virtrolio
+Virtrolio is a webservice that lets you send messages to a friend or colleague's virtual book. It was originally created due to the COVID-19 pandemic closing schools and preventing students from signing each other's yearbooks. Link to the website: https://virtrolio.web.app/
```

Comment:
```suggestion
Virtrolio was founded due to the COVID-19 pandemic closing schools and preventing students from signing each other's yearbooks. Virtrolio is a web service that lets you privately send messages to a friend or colleague and allows only them to view the messages that are destined for them. With the help of some HTML, CSS, Typescript, love, sweat, and tears, we've combined the fun, privacy, and intimacy of yearbook signing with online features such as image embedding and more! 

Check it out! All you need is a Google Account: https://virtrolio.web.app/
```

Reply:
The link to the website should be at the very top, before the first section.

Reply:
I think it makes more sense in context if it comes afterwards